lib/DDG/Spice/Statista.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::Statista;
+# ABSTRACT: Returns statistics from statista.com.
+
+use strict;
+use DDG::Spice;
+
+primary_example_queries "statistics about soccer";
+description "Shows example statistics";
+name "Statista";
+category "facts";
+
+my $api_key = '5db81c2a174170dac75620a16f553dd9';
+my $limit = 16;
+my $lang = 2;
+my @triggers = share('triggers.txt')->slurp;
+
+triggers any => @triggers;
+
+spice to => 'http://api.statista.com/searchJson/apiKey/'.$api_key.'/q/$1/sort/0/lang/'.$lang.'/limit/'.$limit.'/datefrom/0/dateto/0';
```

Comment:
This way we load the key from the environment so it doesn't get compromised by defining it in the actual code :)